ambry-commons/src/main/java/com/github/ambry/commons/CommonUtils.java

```diff
@@ -55,10 +50,6 @@ public static HelixPropertyStore<ZNRecord> createHelixPropertyStore(String zkSer
    */
   public static HelixPropertyStore<ZNRecord> createHelixPropertyStore(String zkAddress, String rootPath,
       List<String> subscribedPaths) {
```

Comment:
And a check here

>     if (zkAddress == null || zkAddress.isEmpty() || rootPath == null) {
      throw new IllegalArgumentException("Invalid arguments, cannot create HelixPropertyStore");
    }

Reply:
Chroot path can be null.

Reply:
can be or cannot be ?